core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -1549,6 +1549,22 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.MASTER)
           .build();
+  public static final PropertyKey MASTER_EMBEDDED_JOURNAL_FLUSH_SIZE_MAX =
+      new Builder(Name.MASTER_EMBEDDED_JOURNAL_FLUSH_SIZE_MAX)
+          .setDefaultValue("64MB")
+          .setDescription("The maximum size in bytes of journal entries allowed "
+              + "in concurrent journal flushing (journal IO to standby masters "
+              + "and IO to local disks.")
+          .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
+          .setScope(Scope.MASTER)
+          .build();
+  public static final PropertyKey MASTER_EMBEDDED_JOURNAL_JOURNAL_SIZE_MAX =
```

Comment:
Do we need two different properties? They looks quite similar in name and description.

Reply:
The two are different, the journal size max is single journal entry max. The journal flush size max is the total journal entries size. Any advice for the naming?